js/models/conversations.js

```diff
@@ -245,6 +246,8 @@
       this.messageCollection.forEach(m => m.trigger('change'));
     },
     async acceptFriendRequest() {
+      // Friend request message conmfirmations (Accept / Decline) are always
```

Comment:
This comment isn't true. Friend request acceptance happens on a per-device basis (because friend request status is device based and not user based). Also, declining a message doesn't send anything.